src/pretalx/submission/migrations/0034_merge_20190304_0958.py

```diff
@@ -0,0 +1,14 @@
+# Generated by Django 2.1.7 on 2019-03-04 09:58
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
```

Comment:
Just a heads-up, I'll probably remove your migrations before the merge, since there's no real reason to keep merge migrations.